Month1/Week2/Section1/Exercise4/Main4.cpp

```diff
@@ -79,10 +81,10 @@ class Printer
 };
 int main() 
 {
-	Printer string;
+	FPrinter string;
```

Comment:
Why is your printer index named `string`? `string` implies that your object is an `std::string` not an `FPrinter`. Also, please follow PascalCase 👀

Reply:
Renamed